cosmos_transfer1/diffusion/config/training/experiment/ctrl_7b_tp_sample_av.py

```diff
@@ -0,0 +1,269 @@
+from cosmos_transfer1.diffusion.config.base.data import get_sampler
+from cosmos_transfer1.diffusion.training.models.model_ctrl import VideoDiffusionModelWithCtrl, ShortVideoDiffusionModelWithCtrl  # this one has training support
+from cosmos_transfer1.diffusion.training.networks.general_dit_video_conditioned import VideoExtendGeneralDIT
+from cosmos_transfer1.diffusion.training.networks.general_dit import GeneralDIT
+from cosmos_transfer1.diffusion.inference.inference_utils import default_model_names
+from cosmos_transfer1.checkpoints import COSMOS_TRANSFER1_7B_CHECKPOINT, COSMOS_TRANSFER1_7B_SAMPLE_AV_CHECKPOINT
+from cosmos_transfer1.diffusion.datasets.example_transfer_dataset import ExampleTransferDataset, AVTransferDataset
+
+
+cs = ConfigStore.instance()
+
+num_blocks = 28
+num_control_blocks = 3
+ckpt_root = '/lustre/fsw/portfolios/nvr/users/tianshic/cosmos_ckpts'#"/mnt/scratch/cache/imageinaire/"   #"checkpoints
+data_root = '/lustre/fs12/portfolios/nvr/users/tianshic/jobs/edify_video4/alpamayo_finetune_debug/driving_FT_7Bv312_lvg_1to6_cameras_multi_camera_005_002_frame_repeat_dbg_2_nodes_1_202504231445/cosmos-predict/cosmos-av-sample-toolkits/datasets/waymo_transfer1/'
+
+def make_ctrlnet_config(
+    hint_key: str = "control_input_segmentation",
+    num_control_blocks: int = 3,
+    pretrain_model_path: str = "",
+    t2v: bool=True,
+    num_frames=121
+) -> LazyDict:
+
+    if pretrain_model_path == "":
+        if t2v:
+            job_name = f"CTRL_7Bv1pt3_t2v_{num_frames}frames_{hint_key}_block{num_control_blocks}_pretrain"
+            job_project = "cosmos_transfer1_pretrain"
+        else:
+            job_name = f"CTRL_7Bv1pt3_lvg_{num_frames}frames_{hint_key}_block{num_control_blocks}_pretrain"
+            job_project = "cosmos_transfer1_pretrain"
+    else:
+        if t2v:
+            job_name = f"CTRL_7Bv1pt3_t2v_{num_frames}frames_{hint_key}_block{num_control_blocks}_posttrain"
+            job_project = "cosmos_transfer1_posttrain"
+        else:
+            job_name = f"CTRL_7Bv1pt3_lvg_{num_frames}frames_{hint_key}_block{num_control_blocks}_posttrain"
+            job_project = "cosmos_transfer1_posttrain"
+    example_multiview_dataset_waymo = L(AVTransferDataset)(
+        dataset_dir="/home/tianshic/code/cosmos-predict1/cosmos-av-sample-toolkits/waymo_apr25_transfer1",
+        num_frames=num_frames,
+        hint_key=hint_key,
+        resolution="720",
+        view_keys=["front"],
+        sample_n_views=-1,
+    )
+    # dataset = L(ExampleTransferDataset)(
+    #     "/home/tianshic/code/cosmos-predict1/cosmos-av-sample-toolkits/datasets/waymo/",
+    #     hint_key,
+    #     121,
+    #     "720",
+    #     True
+    # ),
+    # dataset = L(ExampleTransferDataset)(
+    #     dataset_dir="/home/tianshic/code/cosmos-predict1/cosmos-av-sample-toolkits/datasets/waymo/",
+    #     hint_key = hint_key,
+    #     num_frames = 121,
+    #     resolution = "720",
```

Comment:
If leftover let's remove. If not leftover, let's add a comment to explain the purpose and how to use.